packages/auth/src/auth.ts

```diff
@@ -76,37 +75,9 @@ class Auth implements IAuthMiddleware, TokenEncryption, pluginUtils.IBasicAuth {
     let plugins = (await this.loadPlugin()) as pluginUtils.Auth<unknown>[];
 
     debug('auth plugins found %s', plugins.length);
-    if (!plugins || plugins.length === 0) {
-      plugins = this.loadDefaultPlugin();
-    }
     this.plugins = plugins;
 
-    this._applyDefaultPlugins();
-  }
-
-  private loadDefaultPlugin() {
-    debug('load default auth plugin');
-    const pluginOptions: pluginUtils.PluginOptions = {
-      config: this.config,
-      logger: this.logger,
-    };
-    let authPlugin;
-    try {
-      authPlugin = new HTPasswd(
```

Comment:
I am thinking about this, I don't want to remove this by default, I think make things easier for simple configuration files. Actually happens the same behaviour with the storage it is embebed by default. 

If the goal is disable it, I'd be fine to skip this part of someone set at `config.yaml`. 

```yaml
auth:
```

Intentionally, but if  `auth:` is compeltely missing then go with default. 

Anyhow, since this is an expected behavior would be a breaking change and could not go to 6.x.